src/SFA.DAS.AANHub.Api/Controllers/MembersController.cs

```diff
@@ -94,4 +98,35 @@ public async Task<IActionResult> GetMember(Guid memberId)
 
         return GetResponse(response);
     }
+
+    [HttpPost]
+    [Route("{memberId}/remove")]
+    [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(NoContentResult), StatusCodes.Status204NoContent)]
+    [ProducesResponseType(typeof(NotFoundResult), StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> PostMemberStatus([FromHeader(Name = Constants.RequestHeaders.RequestedByMemberIdHeader)] Guid requestedByMemberId, [FromRoute] Guid memberId, [FromBody] PostMemberRemoveModel request, CancellationToken cancellationToken)
+    {
+        PostMemberStatusCommand command = new()
+        {
+            MemberId = memberId,
+            RequestedByMemberId = requestedByMemberId,
+            Status = request.Status
+        };
+
+        var response = await _mediator.Send(command, cancellationToken);
+
+        if (response.Result is { IsSuccess: false }) return NotFound();
+        if (response.IsValidResponse) return NoContent();
+
+        return new BadRequestObjectResult(FormatErrors(response.Errors));
```

Comment:
GetPostResponse should do this right?

Reply:
GetPostResponse is wired to return a 201 (created) rather than a 204, hence I went with this